CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- ref: Bump main Node.js version to the earliest LTS v18 ([#793](https://github.com/getsentry/sentry-wizard/pull/793))
```

Comment:
l: Feel free to ignore but we started marking breaking changes with a `!` in the JS SDK repo recently and I think that's a pretty nice pattern. Just wanna avoid that this gets released as something other than a major release :D 

```suggestion
- ref!: Bump main Node.js version to the earliest LTS v18 ([#793](https://github.com/getsentry/sentry-wizard/pull/793))
```

Reply:
Happy to do that. If we're okay with a major release though, I'd rather jump to Node 20. WDYT?